src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -1745,24 +1760,36 @@ void VppConfigManager::doCpePolicerConfig (VppClient &vppClient,
           VLOG (1) << "Remove old policer " << policerConfig.name;
           policerConfig.isAdd = 0; // Delete
           vppClient.addDelPolicer (policerConfig, policerIndex);
+
+          if (!isAdd)
+            {
+              continue;
+            }
```

Comment:
This looks unnecessary. The only remaining code inside this loop iteration is already inside an `if (isAdd)`